libflux/src/core/semantic/convert.rs

```diff
@@ -138,6 +142,97 @@ fn convert_builtin_statement(stmt: ast::BuiltinStmt, fresher: &mut Fresher) -> R
     })
 }
 
+pub type MonoTypeMap = SemanticMap<String, MonoType>;
```

Comment:
Similarly can you re-use the MonoTypeMap from `types.rs`?

Reply:
✅